wsd-frontend/src/components/wsd/MemeComment/MemeComment.tsx

```diff
@@ -114,6 +167,14 @@ export function MemeComment({ comment }: { comment: Includes<APIType<'PostCommen
             >
               <Icons.ArrowBigDown size={20} className={cn(feedback === -1 && 'text-destructive fill-destructive')} />
             </Button>
+            <Button
+              onClick={handleShare}
+              size="sm"
+              className="flex items-center gap-1 p-2 rounded-md transition-colors text-gray-500 hover:bg-secondary bg-transparent"
```

Comment:
Can we keep the colors consistent with other buttons? Also design wise do you think we should remove the hover background like upvote and downvote buttons?